### DISCUSSION TITLE
Importing small molecule PDB connects atoms, extra bonds, that I do not want

### DISCUSSION THREAD

**Discussion by AqueousSpacious** [BradyAJohnston/MolecularNodes#478] [Q&A]
Hi,

What a wonderful tool you have created for the structural biology community! Thank you for all the hard work.
Molecular nodes is not made for small molecules, so this makes sense if a rare edge case.

Observation of problem:
1. Downloaded Rhoadmine 800 from pubchem as the 3D conformer SDF (https://pubchem.ncbi.nlm.nih.gov/compound/127750#section=2D-Structure). This opens well in blender no problem.
2. I wanted to edit a bond length, so I imported the SDF into pymol, created an additional state with the new bond length (to show animation of vibrating bond by interpolating between state 1 & 2) and then exported the PDB
3. The result is that there is an additional bond between Oxygen 1 and C 17 or something when loaded into blender (but not when opened in pymol again)

Work around I have tried:
a. putting the file into open Babel to convert to PDB then working on it (open babel destroys the original file bond lengths and angels for some reason)
b. other file formats like .cif or .sdf are opened by molecular nodes, but they d o not seem to have access to the states information?
c. deconnecting bonds but I have not figured out the magic fix for this yet.

Any help would be immensely appreciated, thank you!
Michael

![BondProblem](https://github.com/BradyAJohnston/MolecularNodes/assets/165734865/6cd9ddf0-da75-4e58-a3f8-b0712dc382c3)

**Comment by AqueousSpacious** [comment]
Figured out a solution:
-use the Topology Bonds Find node and plug into Style Sticks node
![BondSolution](https://github.com/BradyAJohnston/MolecularNodes/assets/165734865/774905ea-f7e7-4f27-a375-350915ce420c)
![BondSolutionNodes](https://github.com/BradyAJohnston/MolecularNodes/assets/165734865/0bf6cdea-a0da-4724-8558-0329524c4c70)

**Reply by BradyAJohnston** [reply]
Glad it's working! It's still something worth looking into, to and improve the import so I'll create an issue from it.
